home.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:io';
+import 'App.dart';
+
+void main() {
+  Home().start();
+}
+class Home {
+  start() {
+    print("Welcome to the Home page");
+    print("--------------------------");
+    print("Choose one by entering its number:");
+    List options = ["1.Sign up", "2.Login", "3.Exit"];
+    options.forEach((element) {
+      print(element);
+    });
+    String? number = stdin.readLineSync();
+    if (number == '1') {
+      App().runHome(UserOptions.signUp);

```

Comment:
instead of calling App().run which creates a new app instance!! too many instances will stay in the memory for nothing.

App should create a home and home.start() should tell the app what to do next in a returned value or a callback or a stream / future (will talk about this later).